File: mario.py
from cs50 import get_int

height = 0
#pbtain the height of the required pyramid
while height < 1 or height > 8:
    height = get_int("Height: ")

#printout the pyramid
for i in range(0, height):
    print(" " * (height - 1 - i) + "#" * (i + 1) + "  " + "#" * (i + 1))

File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h = 0;
    while (h <= 0 || h > 8)
    {
        h=get_int("Height :");
    }
    int n=1;
    do
    {
        for(int y=0; y<h-n;y++)
        {
            printf(" ");
        }
        for(int y=1; y<=n; y++)
        {
            printf("#");
        }
        printf("  ");\
        for(int y=1; y<=n; y++)
        {
            printf("#");
        }
        printf("\n");

        n++;
    }while(n<=h);


}